scripts/executors/src/start.ts

```diff
@@ -218,8 +230,9 @@ async function getSelectedTarget(
     name: 'target',
     message: 'Select target to run',
     choices: targetChoices,
+    initial: preselectedTargetChoice,
     suggest,
-    limit: 5,
+    limit: targetChoices.length,
```

Comment:
we wanna show all possible project targets at once - it goes hand in hand with default target selection in order to not render weird UI where pre-selected target could be hidden